casingSimulations/base.py

```diff
@@ -12,7 +13,7 @@ class LoadableInstance(properties.Instance):
 
     def validate(self, instance, value):
         if isinstance(value, str):
-            return value
+            value = load_properties(value)
```

Comment:
This will load the file on validate (then validate the loaded instance), rather than saving the string to be dealt with later.